drivers/base/bootvid/common.c

```diff
@@ -15,8 +15,61 @@ ULONG VidpScrollRegion[4] =
     SCREEN_HEIGHT - 1
 };
 
+const PALETTE_ENTRY DefaultPalette[BV_MAX_COLORS] =
+{
+    {0x00, 0x00, 0x00}, // Black
+    {0x00, 0x00, 0x80}, // Blue
+    {0x00, 0x80, 0x00}, // Green
+    {0x00, 0x80, 0x80}, // Cyan
+    {0x80, 0x00, 0x00}, // Red
```

Comment:
```suggestion
    {0x00, 0x00, 0x80}, /* Blue */
```

Reply:
Oh I just noticed you suggested to change color data along with comments. I tried changing this value, but it doesn't change BSOD color, so my assumption that default palette gets changed is correct. Also see this reference: https://en.wikipedia.org/wiki/Color_Graphics_Adapter#With_an_RGBI_monitor

Reply:
That's fine because in your InitializePalette (used when the reset display is called) you manually exchanged the components.

Reply:
<details>
  <summary>No, bootvid from current branch</summary>
  
![bug](https://user-images.githubusercontent.com/37072976/81506100-e4c60380-9315-11ea-84de-dd5616a3f830.png)
</details>

Reply:
I like this RSOD <3